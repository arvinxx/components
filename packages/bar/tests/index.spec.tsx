import React from 'react';
import renderer from 'react-test-renderer';

import Bar from '@arvinxu/modules-bar';

test('Bar', () => {
  const component = renderer.create(<Bar />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
