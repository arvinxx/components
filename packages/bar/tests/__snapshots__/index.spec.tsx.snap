// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bar 1`] = `
<div>
  World!
</div>
`;
